extensions/amazon-lambda-rest/runtime/src/main/java/io/quarkus/amazon/lambda/http/DefaultLambdaSecurityIdentityProvider.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkus.amazon.lambda.http;
+
+import java.security.Principal;
+import java.util.Map;
+
+import io.quarkus.amazon.lambda.http.model.AwsProxyRequest;
+import io.quarkus.amazon.lambda.http.model.AwsProxyRequestContext;
+import io.quarkus.security.identity.SecurityIdentity;
+
+public class DefaultLambdaSecurityIdentityProvider implements LambdaSecurityIdentityProvider {
```

Comment:
With the changes I have suggested this would become an CDI bean that implements `IdentityProvider`.